deployment/src/test/java/io/quarkiverse/univocityparsers/deployment/models/CSVFileExample.java

```diff
@@ -0,0 +1,241 @@
+/*
+ *  Copyright (C) DALKIA SA - 2015-2017 All Rights Reserved
+ *  Unauthorized copying of this file, via any medium is strictly prohibited
+ *  Proprietary and confidential
```

Comment:
I'm pretty sure you don't want to put this in something Open Source.

Reply:
Obviously ! grrr Intellij for hidden to me